backend/services/userService.js

```
@@ -0,0 +1,210 @@
+    }
+}
+// Find All Users who have joined a specific event
+async function findByEventsJoined(eventId) {
+    // since events are not implemented yet, this function will not do anything
+    try
+    {
+        throw new Error("Events are not implemented yet");
+        if(!eventId)
+        {
+            throw new Error("eventId is required");
+        }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)